xlive/Blam/Engine/render/render.h

```diff
@@ -65,8 +71,8 @@ struct window_bound
 	int32 single_view;
 	int32 window_bound_index;
 	int32 user_index;
-	s_camera render_camera;
-	s_camera rasterizer_camera;
+	struct render_camera render_camera;
+	struct render_camera rasterizer_camera;
```

Comment:
? Struct struct?

Reply:
Intended, does not compile unless you use struct in front of it.
These are the original member and struct names.